docs/source/modules/zos_fetch.rst

```diff
@@ -126,13 +126,13 @@ tmp_hlq
 
 
 ignore_sftp_stderr
-  During data transfer through sftp, the module fails if the sftp command directs any content to stderr. The user is able to override this behavior by setting this parameter to ``true``. By doing so, the module would essentially ignore the stderr stream produced by sftp and continue execution.
+  During data transfer through SFTP, the SFTP command directs content to stderr. By default, the module essentially ignores the stderr stream produced by SFTP and continues execution. The user is able to override this behavior by setting this parameter to ``false``. By doing so, any content written to stderr is considered an error by Ansible and will have module fail.
```

Comment:
Last section: will have THE module fail, or will CAUSE THE module TO fail.

Reply:
Thanks Rich, sending this up in a min.

Reply:
Same text was also in zos_copy, fixed both.

Reply:
Done.